### ISSUE TITLE
Bern features: Not possible to change sharing option of categories in the tab "Mine"

### ISSUE THREAD

**Issue by dagraf** [opencast/annotation-tool#450]
Clicking on the three possible icons is possible. But has no effect:
- When I try to change to public: Shown icon shortly switches to "globe" and then back to the original setting ("privat" or "shared with administrator").
- When I try to chang to privat or shard-with-administrator: Nothing at all happens.

**Comment by dagraf** [comment]
Also: Tool tip when hovering over the set option is not shown. It says "timeline.change myvisibility.tooltip"

**Comment by Arnei** [comment]
I currently cannot reproduce the issue on my local installation (which should be up-to-date with your installation). I'll have to talk with Julian about this.

Also fixed the missing tooltip.

**Comment by JulianKniephoff** [comment]
Yeah, that was partly an oversight on my side while merging together Arne's PRs for the SWITCH deployment, and partly a bug in #407 that apparently only surfaced with (the recent changes to) #414.

Anyhow, I just deployed an updated version to the SWITCH staging system, @dagraf, and @Arnei will get another PR for his PR later today or tomorrow. ;)

**Comment by dagraf** [comment]
Works now as expected. Thx!

**Comment by dagraf** [comment]
Unfortunately, this bug does appear again. On Staging and Production.

**Comment by JulianKniephoff** [comment]
Weird! :thinking: I'll look into it.

**Comment by JulianKniephoff** [comment]
@dagraf this should be fixed again. :)

**Comment by dagraf** [comment]
@JulianKniephoff There is some improvement but still a bug: I can now switch the visibility setting only once. For example when I switch it from private to public I can then make no other change (i.e. switch back or to "shared with administrator"). After a reload of the browser I can again change the visibility setting, but again only once.

**Comment by JulianKniephoff** [comment]
Oof, that's weird! I will take another look as soon as I can. :(

**Comment by dagraf** [comment]
Weird, indeed. Because this bug does only appear in the first category I created after the hotfix. In all other cateogries it works fine. So pleases close it for now. I will have an eye on this an reopen it if I can reproduce it. Thx!

**Comment by JulianKniephoff** [comment]
Alright. :)
